plotview.h

```diff
@@ -71,7 +71,7 @@ public slots:
     size_t zoomSample;
 
     int fftSize = 1024;
-    int zoomLevel = 0;
+    int zoomLevel = 1;
```

Comment:
I changed this to 1 because 0 is invalid and causes `PlotView::samplesPerColumn()` to crash due to a division by zero.